src/yb/master/master_main.cc

```diff
@@ -69,6 +71,14 @@ static int MasterMain(int argc, char** argv) {
   // Reset some default values before parsing gflags.
   FLAGS_rpc_bind_addresses = strings::Substitute("0.0.0.0:$0", kMasterDefaultPort);
   FLAGS_webserver_port = kMasterDefaultWebPort;
+
+  string host_name;
+  if (GetHostname(&host_name).ok()) {
+    FLAGS_metric_node_name = (host_name) + ":" + std::to_string(kMasterDefaultWebPort);
```

Comment:
Minor: you can use `strings::Substitute("$0:$1", host_name, kMasterDefaultWebPort)` as you can see from the FLAGS_rpc_bind_addresses example above.

Reply:
I'll fix this soon. Thanks.